semantic/carla_controller/transform_dataset.py

```diff
@@ -1,36 +1,18 @@
-import carla
+from semantic.carla_controller.labels import CARLA_SEMANTIC_COLORS
+
 import cv2
 import os
 
 import numpy as np
 from pathlib import Path
 from random import randint
 from shutil import copy
-from typing import Dict
 
 
 SOURCE_DIRECTORY = "./output"
 DESTINATION_DIRECTORY = "./dataset"
 
 
-def load_dataset_stats():
```

Comment:
Why are we removing this?  We aren't using it?

Reply:
No, we never coded it. The original idea was that we could build a stats generation of the dataset, such that we could organize our training dataset to have higher numbers of certain scenarios/objects. I abandoned it once I realized that we have a limited dataset and can't afford to do this at the moment, and we expanded our category count anyway.